app/controllers/cart_items_controller.rb

```diff
@@ -1,12 +1,15 @@
 class CartItemsController < ApplicationController
   def create
     @item = Item.find(params[:item_id])
-    @cart.add_item(@item.id)
+    if params[:subtract]
+      @cart.remove_item(@item.id)
+      flash[:danger] = "#{@item.title} removed from cart!"
+    else
+      @cart.add_item(@item.id)
+      flash[:success] = "#{@item.title} added to cart!"
+    end
```

Comment:
Think about a Cart method that can update and will take in the Params and the @item